source/common/common/assert.h

```diff
@@ -13,6 +16,46 @@ class ActionRegistration {
 };
 using ActionRegistrationPtr = std::unique_ptr<ActionRegistration>;
 
+/*
+ * EnvoyBugStackTrace captures and writes the stack trace to Envoy Bug
+ * to assist with getting additional context for reports
+ */
+class EnvoyBugStackTrace : Logger::Loggable<Logger::Id::envoy_bug> {
```

Comment:
declare inheritance visibility you want vs having implicit

Reply:
Done